info.limpet/src/info/limpet/data/persistence/xml/XStreamHandler.java

```diff
@@ -62,7 +62,14 @@
 		
 		
 		xstream.alias("Point", org.geotools.geometry.iso.primitive.PointImpl.class);
```

Comment:
@pecko - for the Point handling, I hope we can just store [x,y] coordinate pairs.  We can use our GeoSupport class to convert them to/from PointImpl objects.  Then we won't need any of these other geospatial metadata fields.